src/commands/status/index.ts

```diff
@@ -48,7 +50,12 @@ export default class ShowStatus extends Base {
         const loggedInOrg = this.repoConfig?.org || this.userConfig?.org
         if (this.hasToken()) {
             if (loggedInOrg) {
-                this.writer.successMessage(`Currently logged in to DevCycle as a member of ${loggedInOrg.display_name}`)
+                this.writer.successMessage('Logged into DevCycle with:')
+                this.writer.showRawResults(chalk.green(`\tOrganization: ${loggedInOrg.display_name}`))
+                const loggedInProject = this.repoConfig?.project || this.userConfig?.project
```

Comment:
Same thing here — it seems like we were already duplicating the repo/user config logic in a few places, but if you look at the Base command the project and org can be defined in more ways than just the configs. We should rely on these properties to tell us which project/org is the current one
```
this.projectKey = flags['project']
            || process.env.DEVCYCLE_PROJECT_KEY
            || process.env.DVC_PROJECT_KEY
            || this.repoConfig?.project
            || this.userConfig?.project
            || ''
this.orgId = flags['org'] || this.repoConfig?.org?.id || this.userConfig?.org?.id
```

Reply:
looks like there's no `org` flag on the base class so we should drop the `flags['org']` in
```
this.orgId = flags['org'] || this.repoConfig?.org?.id || this.userConfig?.org?.id
```
Also I don't think letting users pass an org as flag makes sense since trying to use an org id other than the one that you have authorized with would result in auth errors.

Let me know if you think otherwise

Reply:
The org flag is used by any command that extends the AuthCommand, it will use the specified org for authentication rather than org in the config (ie. `login again`, `login sso`, `organizations select`, `projects select`, `repo init`)